webapp/src/services/index.ts

```diff
@@ -67,7 +67,7 @@ const azureDevOpsPluginApi = createApi({
         [Constants.pluginApiServiceConfigs.getSubscriptionList.apiServiceName]: builder.query<SubscriptionDetails[], FetchSubscriptionList>({
             query: (params) => ({
                 headers: {[Constants.common.HeaderCSRFToken]: Cookies.get(Constants.common.MMCSRF)},
-                url: Constants.pluginApiServiceConfigs.getSubscriptionList.path,
+                url: `${Constants.pluginApiServiceConfigs.getSubscriptionList.path}/${params.team_id}`,
```

Comment:
here we are sending the team id in path param as well as in body do we need to send this twice